examples/scala/build.sbt

```diff
@@ -169,3 +169,26 @@ lazy val root = (project in file("."))
     ),
     java17Settings
   )
+
+lazy val connect = (project in file("./connect"))
+  .settings(
+    run / fork := true,
+    name := "example",
+    scalaVersion := scala213,
+    mainClass := Some("example.DeltaConnect"),
+    libraryDependencies ++= Seq(
+      "io.delta" %% "delta-connect-client" % "4.0.0",
+      "org.apache.spark" %% "spark-connect-client-jvm" % "4.0.0"
+    ),
+    extraMavenRepo,
+    resolvers ++= Seq(
+      Resolver.mavenLocal,
+      "Delta" at "https://oss.sonatype.org/content/repositories/iodelta-1229"
+    ),
```

Comment:
remove this this will be passed in as `extraMavenRepo` when run right? this means we don't need to update this when staging repo changes